### ISSUE TITLE
UI breaks when font is increased from device settings

### ISSUE THREAD

**Issue by AnttiRumpunen** [fmidev/weather-app#933]
![Image](https://github.com/user-attachments/assets/f49c5980-d853-4122-9c8b-cf96ef0f64b4)

I believe user has increased font size => Feels like etc panel is not visible. Device is Samsung Galaxy A22.
